app/api-v1/routes/recipe.js

```diff
@@ -35,13 +35,13 @@ module.exports = function (recipeService, identityService) {
       const { address: supplierAddress } = await identityService.getMemberByAlias(req, supplier)
 
       const recipe = await recipeService.createRecipe({
-        externalId,
+        external_id: externalId,
```

Comment:
I don't like the casing changes. Why?

Reply:
because in the database we should follow same pattern. And we started using snake casing so created thisi mapping for only db methods. For APIs and JS world we should use camel casing. In the api-doc it remains camel case and we expect that format, trying to keep consistent